scripts/workspace-conversion/README.md

```diff
@@ -0,0 +1,61 @@
+| 5 | Data Use Limitation | `library:dataUseRestriction` | `TerraDCAT_ap:hasDataUsePermission.0`<br>Not a perfect mapping, we will need to normalize "General Research Use" to "TerraCore:NoRestriction", preferably using the mapping found [here](https://github.com/DataBiosphere/terra-ui/blob/dev/src/pages/library/dataBrowser-utils.js#23) under `datasetReleasePolicies` |
+| 6 | Cohort Phenotype/Indication | `library:indication`<br>ex: "Mesothelioma" | kathy: might not be useful, could consider pre-pending with source (ie: "TCGA" or "Anvil_phenotype") |
+| 7 | Cohort Name | `library:datasetName` | `dct:title` |
+| 8 | Dataset Version | `library:datasetVersion` | `dct:version` |
+| 9 | Cohort Description | `library:datasetDescription` | `dct:description` |
+| 10 | Dataset Owner | `library:datasetOwner`<br>ex: "NCI" | `TerraDCAT_ap:hasDataCollection.0.dct:identifier`<br>If we have enough information, we can also build:<br> `TerraDCAT_AP:hasDataCollection.0.dct:publisher: National Cancer Institute`<br>`TerraDCAT_AP:hasDataCollection.0.dct:title: National Cancer Institute` |
+| 11 | Dataset Custodian | `library:datasetCustodian`<br>ex: "dbGAP" | `TerraDCAT_ap:hasDataCollection.0.dct:identifier`<br>Note: This conflicts with "Dataset Owner", but I dont think we have another option for where to put this. |
+| 12 | Dataset Depositor | `library:datasetDepositor` | `contributors.0.contactName`<br>`contributors.0.correspondingContributor = true` |
+| 13 | Contact Email | `library:contactEmail` | `contributors.0.email` |
+| 14 | Research Institute | `library:institute` | `contributors.0.institution` |
+| 15 | Primary Disease Site | `library:primaryDiseaseSite`<br>ex: "Pleura" | `samples.disease.0`?<br>ex: "Brain Cancer" |
+| 16 | Project Name | `library:projectName`<br>ex: "TCGA" |  |
+| 17 | Genome Reference Version | `library:reference`<br>ex: "GRCh37/hg19" |  |
+| 18 | Data File Formats | `library:dataFileFormats`<br>ex: "TXT, MAF" | `files.0.dcat:mediaType`<br>`files.0.count = 0`<br>`files.0.byteSize = 0`<br>Note: No way of knowing how many files match each file format |
+| 19 | Profiling Instrument Type | `library:technology` |  |
+| 20 | Profiling Protocol | `library:profilingProtocol` |  |
+| 21 | Depth of Sequencing Coverage (Average) | `library:coverage` |  |
+| 22 | Study Design | `library:studyDesign`<br>ex: "Tumor/Normal" | `samples.disease.0`<br>Note: concat value with "Primary Disease Site"? |
+| 23 | Cell Type | `library:cellType`<br>ex: "Primary tumor cell, Whole blood" | `samples.disease.0`<br>Note: concat value with "Primary Disease Site"? |
+| 24 | Reported Ethnicity | `library:ethnicity` |  |
+| 25 | Cohort Country of Origin | `library:cohortCountry`<br>ex: "USA" |  |
+| 26 | Requires External Approval | `library:requiresExternalApproval` |  |
```

Comment:
This and structured data use limitations should probably be addressed by mapping to Data Use ontology data use modifier terms but it depends upon how they're used in the data.  I'm happy to try to help but Jonathan Lawson, Greg Rushton would be helpful here too.